packages/overlay/test/focus-trap.test.js

```diff
@@ -1,167 +1,76 @@
-    }
-  },
-);
+  function getFocusedElementIndex() {
+    return focusableElements.findIndex(isElementFocused);
+  }
 
-customElements.define(
-  'overlay-components-wrapper',
-  class extends PolymerElement {
-    static get template() {
-      return html`
-        <vaadin-overlay id="defaultOverlay" focus-trap>
-          <template>
-            overlay-content
+  describe('focusable elements', () => {
+    beforeEach(async () => {
+      overlay = fixtureSync('<vaadin-overlay focus-trap></vaadin-overlay>');
+      await nextRender();
+      overlayPart = overlay.$.overlay;
+      overlay.renderer = (root) => {
+        if (!root.firstChild) {
+          root.innerHTML = `
             <button>tabindex 0</button>
             <button tabindex="-1">tabindex -1</button>
             <select tabindex="2">
               <option>tabindex 2</option>
             </select>
-            <vaadin-text-field></vaadin-text-field>
             <textarea tabindex="1">tabindex 1</textarea>
             <input type="text" id="text" value="tabindex 0" />
-            <vaadin-radio-group>
-              <vaadin-radio-button id="radioButton1" label="Button 1"></vaadin-radio-button>
-              <vaadin-radio-button id="radioButton2" label="Button 2"></vaadin-radio-button>
-              <vaadin-radio-button id="radioButton3" label="Button 3"></vaadin-radio-button>
-            </vaadin-radio-group>
-            <vaadin-button>tabindex 0</vaadin-button>
-          </template>
-        </vaadin-overlay>
-        <vaadin-overlay id="shadowOverlay" focus-trap>
-          <template>
-            <container-with-shadow></container-with-shadow>
-            <input type="text" id="text" />
-          </template>
-        </vaadin-overlay>
-        <vaadin-overlay id="emptyOverlay" focus-trap>
-          <template></template>
-        </vaadin-overlay>
-      `;
-    }
-  },
-);
-
-customElements.define(
-  'nested-overlay-wrapper',
-  class extends PolymerElement {
-    static get template() {
-      return html`
-        <vaadin-overlay id="outer" focus-trap>
-          <template>
-            <button>Button</button>
-            <vaadin-overlay id="nested">
-              <template>Inner content</template>
-            </vaadin-overlay>
-          </template>
-        </vaadin-overlay>
-      `;
-    }
-  },
-);
-
-describe('focus-trap', () => {
-  let overlay, parent, overlayPart, focusableElements;
-
-  function getFocusedElementIndex() {
-    return focusableElements.findIndex(isElementFocused);
-  }
-
-  describe('in a component scope', () => {
-    beforeEach(async () => {
-      parent = fixtureSync('<overlay-components-wrapper></overlay-components-wrapper>');
-      await nextRender();
-      overlay = parent.$.defaultOverlay;
+          `;
+        }
+      };
       overlay.requestContentUpdate();
-      overlayPart = overlay.$.overlay;
-      focusableElements = getFocusableElements(overlay.$.overlay);
-      window.focus();
```

Comment:
Using `window.focus()` was only needed with the old test runner, we don't need it with WTR.